src/kontrol/prove.py

```diff
@@ -869,9 +869,10 @@ def _init_cterm(
             # Symbolic account for the contract being executed
             accounts.append(Foundry.symbolic_account(Foundry.symbolic_contract_prefix(), contract_code))
         else:
+            storage_map = KVariable(Foundry.symbolic_contract_prefix() + '_STORAGE', sort=KSort('Map'))
```

Comment:
Is this variable declaration needed here and as a separate argument to `_create_cse_accounts`? It gets initialized in the same manner within `_create_cse_accounts`: https://github.com/runtimeverification/kontrol/blob/fe577e524655da12a6313c3baa4ed671890b23ed/src/kontrol/prove.py#L977
On the first (non-recursive) invocation of `_create_cse_accounts`, `contract_name` is `Foundry.symbolic_contract_prefix()`, so it gets initialized with the exact same values, I think.

Reply:
Fixed.